# fleet_management/actions/FleetManagement.action

# Goal definition
int32 origin_city
int32 destination_city
float32 load_tonnage
---
# Result definition
string[] vehicle_routes
int32 selected_truck
---
# Feedback
int32 estimated_delivery_time
